blog/2021/02/julia-1.6-highlights.md

```diff
@@ -138,7 +138,32 @@ method invalidations](https://julialang.org/blog/2020/08/invalidations/).
 
 ## Compiler latency reduction
 
-_Jeff Bezanson_
+_Jameson Nash and Jeff Bezanson_
+
+Load times and compilation pauses continue to be a significant nuissance for
+Julia users, and one of our main technical challenges. We try to make at least
+a little progress on it in every release. This time, there aren't any major
+breakthroughs, but we do have some modest improvements due to work on the
+method table data structure.
+
+Method specificity is a partial order, and prior to 1.6 we stored methods in
+sorted order. We also attempted to identify ambiguous methods on insertion,
+hoping to avoid repeating the work for each future query.
+Unfortunately, sorting a partial order requires quadratic time, and this time
+began to show up prominently during package loading (when a package's methods
+need to be inserted into the currently-active method tables).
+
+We improved things by making the process lazier, moving sorting and ambiguity
+detection into the algorithm for finding matching methods.
+This algorithm runs very often, so it was not at first intuitive that this
+change would help.
+But the key is that the vast majority of queries are for specific enough
+types that most possible matches can be eliminated easily, leaving
+many fewer inputs to the most expensive steps.
+
```

Comment:
Should https://github.com/JuliaLang/julia/pull/36208 and `Base.Experimental.@compiler_options` be mentioned here as well?